schema_based_on_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patient(
```

Comment:
[OPTIONAL]
- [ ] To be more grammatically accurate you ay consider saying:
```suggestion
CREATE TABLE patients(
```